iceberg/iceberg-handler/src/test/results/positive/use_basic_stats_from_iceberg.q.out

```diff
@@ -396,14 +396,14 @@ Stage-0
     Stage-1
       Reducer 2 vectorized
       File Output Operator [FS_8]
-        Select Operator [SEL_7] (rows=18 width=95)
+        Select Operator [SEL_7] (rows=18 width=192)
           Output:["_col0","_col1","_col2"]
         <-Map 1 [SIMPLE_EDGE] vectorized
           SHUFFLE [RS_6]
-            Select Operator [SEL_5] (rows=18 width=95)
+            Select Operator [SEL_5] (rows=18 width=192)
               Output:["_col0","_col1","_col2"]
-              TableScan [TS_0] (rows=18 width=95)
-                default@tbl_ice,tbl_ice,Tbl:COMPLETE,Col:COMPLETE,Output:["a","b","c"]
```

Comment:
We don't remove the table directory of `tbl_ice`, the col should be stale by design